tests/test/constants.rs

```diff
@@ -105,6 +122,99 @@ fn generic_impl() {
     }
 }
 
+#[test]
```

Comment:
A comment on each goal and what it's purpose is would be helpful.

Reply:
I don't see the rest of the tests doing this, though

Reply:
A lot of tests haven't done this. Recently though, we have been trying to at least make sure *new* tests get comments.

Reply:
I added comments on the new tests

Reply:
Great thanks!